### PR TITLE
Edoxaban

### PR DESCRIPTION
**Pull request by richiecroker**
correct pull request branch

### CODE REVIEW

File: openprescribing/measure_definitions/edoxaban.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  "title": [
+    "Percentage DOACs items not prescribed as edoxaban"
+  ],
+  "description": [
+    "Percentage DOACs items not prescribed as edoxaban"
+  ],
+  "numerator_short": "DOAC items not prescribed as edoxaban",
+  "denominator_short": "DOACs items",
+  "why_it_matters": [
+    "In January 2022 the NHS in England <a href='https://www.england.nhs.uk/wp-content/uploads/2022/01/B1279-national-procurement-for-DOACs-commissioning-recommendations-v1.pdf'>",
```
Comment (brianmackenna) [inline openprescribing/measure_definitions/edoxaban.json:12]: The new contract documents are out so I think we should update this before launch

File: openprescribing/measure_definitions/edoxaban.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    "richard.croker@phc.ox.ac.uk"
+  ],
+  "checked_by": [
+    ""
+  ],
+  "date_reviewed": [
+    "2022-04-11"
+  ],
+  "next_review": [
+    "2023-03-14"
```
Comment (brianmackenna) [inline openprescribing/measure_definitions/edoxaban.json:64]: do we have a method/scipt for systematically identifying when they are ready for review?

**Review by brianmackenna — APPROVED** [review]
Great stuff @richiecroker

File: openprescribing/measure_definitions/edoxaban.json (hunk 2 above)
Comment (richiecroker) [reply openprescribing/measure_definitions/edoxaban.json:65]: Unfortunately not at the moment.  When I did a review of the measure definitions a few years ago I put the dates in in order to have exactly that, but obviously we haven't had the resource to fully implement.  Something to reconsider now.

**Pull request merged**